src/comments.js

```diff
@@ -834,7 +834,33 @@ function printComment(commentPath, options) {
       return "#" + comment.value;
     case "CommentBlock":
     case "Block":
-      return "/*" + comment.value + "*/";
+      const lines = comment.value.split("\n");
+      const isJsDocComment =
+        lines.length > 1 &&
+        lines.slice(0, lines.length - 1).every(line => line.trim()[0] === "*");
+      if (!isJsDocComment) {
+        return "/*" + comment.value + "*/";
+      }
```

Comment:
`addAlignmentToDoc` is an absolute alignment, so it doesn't take into account the current indentation. If you say 6, it's going to put 6 no matter how the indentation is for this element.

Using `addAlignmentToDoc` was the first idea I had (and your code looked like it would be using) but then I thought about something else.

What we can do is to find at what indent the `/*` starts (let's call it `startIndent`) and for each line use `align()` with `lineIndent - startIndent`. This way we only have to make sure that the delta in spaces between the first line and all the lines is correct. We can let the normal algorithm decide what the first indent should be (which it does a good job at).

If we do that, we shouldn't care about reading `*` because all the spaces in the comment will be respected.